librosa/core/constantq.py

```diff
@@ -62,9 +63,9 @@ def cqt(y, sr=22050, hop_length=512, fmin=None, n_bins=84,
```

Comment:
I agree it works as written, my point was readability.  I guess I'm still reeling from wondering what n_filters is going to do (the plot thickens...)